cmd/internal/cli/actions.go

```diff
@@ -81,6 +81,7 @@ func handleLibrary(ctx context.Context, imgCache *cache.Handle, pullFrom, librar
 	c := &scslibrary.Config{
 		AuthToken: authToken,
 		BaseURL:   libraryURL,
+		Logger:    sylog.DebugLogger{},
```

Comment:
Could we use `(golog.Logger)(sylog.DebugLogger{})` and add `golog` dependency in package where it's used rather than in `sylog` because it would be nice for plugin to have `sylog` as a public package